src/users/users.controller.ts

```diff
@@ -9,23 +9,29 @@ import {
   Res,
 } from '@nestjs/common';
 import { UserService } from './users.service';
-import { CreateUsersDto, UpdateUserDto } from 'src/dtos/users.dto';
+import { ApiResponse, ApiTags, ApiParam } from '@nestjs/swagger';
+import { RegistrationDTO, UpdateUserDto } from 'src/users/users.dto';
 
 @Controller('users')
+@ApiTags('User')
 export class UsersController {
   constructor(private readonly usersService: UserService) {}
 
+  @ApiResponse({
+    status: 200,
+    description: 'A user has been succesfully created',
+  })
   @Post('/create-user')
   async createStudent(
```

Comment:
Why are you attaching student to the name of the service? Is it a student app

Reply:
noted boss